- added recipes for Windows PostgreSQL dependencies

ptosco opened the pull request.

greglandrum commented:
Now it certainly attempts to build postgresql, which is a big step forward.
It doesn't actually work for me since it seems that the postgresql 9.4 windows build is looking for an older version of visual studio. This isn't horribly concerning to me since I don't think we particularly need to support every possible config.

I'll hold off on merging though until you have a chance to comment as to whether or not you think this is odd behavior.

ptosco commented:
If I remember correctly I sourced the appropriate vcvars.bat script from the VS version I wished to build PostgreSQL with ahead of building in the relevant shell.
I'll check this later this afternoon.

> On 30 Jun 2017, at 05:09, Greg Landrum <notifications@github.com> wrote:
> 
> Now it certainly attempts to build postgresql, which is a big step forward.
> It doesn't actually work for me since it seems that the postgresql 9.4 windows build is looking for an older version of visual studio. This isn't horribly concerning to me since I don't think we particularly need to support every possible config.
> 
> I'll hold off on merging though until you have a chance to comment as to whether or not you think this is odd behavior.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

ptosco commented:
PostgreSQL <=9.5.2 won't build with VS2015. When I build 9.4.4 it worked for me as I have VS2013 and VS2015 on my Windows laptop and I hadn't even noticed that it had picked VS2013. I have added a patch such that PostgreSQL >=9.4 and <=9.5.2 builds with VS2015 and updated meta.yaml accordingly.
